tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -348,7 +348,7 @@ function configure_crio() {
 
 
 	if [ "${DEBUG}" == "true" ]; then
-		cat <<EOF | tee -a $crio_drop_in_conf_file_debug
+		cat <<EOF | tee $crio_drop_in_conf_file_debug
```

Comment:
It seems this falls into the `just ensure we remove any exist configuration` category. What about making this fix even more explicit with an `rm -f "op_in_conf_file_debug"` at https://github.com/kata-containers/kata-containers/pull/9535/commits/bde3773c1473bde771d4fb4da9e440be284cfac7#diff-08a680425cfe93e69c4108dc8fd91a9aeaefbf016bd9941b03c66834bf87a341R344 ?